src/fastboot-info.js

```diff
@@ -7,13 +7,15 @@ var FastBootResponse = require('./fastboot-response');
  * current HTTP request from FastBoot. This is injected
  * on to the FastBoot service.
  */
```

Comment:
Should this document the constructor arguments?  Definitely non-blocking, but it might be nice to document what things are expected to be in `infoOptions`...